src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -1,25 +1,30 @@
 package org.broadinstitute.dsde.rawls.graph
 
+import java.util.concurrent.atomic.AtomicBoolean
 import java.util.logging.{LogManager, Logger}
 
-import com.tinkerpop.blueprints.impls.orient.{OrientBaseGraph, OrientGraph}
+import com.tinkerpop.blueprints.impls.orient.OrientGraph
+import org.broadinstitute.dsde.rawls.dataaccess.{DataSource, RawlsTransaction}
 import org.scalatest.BeforeAndAfterAll
 
 trait OrientDbTestFixture extends BeforeAndAfterAll {
   this : org.scalatest.BeforeAndAfterAll with org.scalatest.Suite =>
 
-  var graph: OrientBaseGraph = _
   val testDbName: String
+  lazy val dataSource = DataSource("memory:"+testDbName, "admin", "admin")
+  lazy val graph = new OrientGraph("memory:"+testDbName)
```

Comment:
why is there both a datasource/rawlstransction and a graph?  the test fixture should probably only use the graph directly or use our functionality, depending on what we intend to test with this.  Fine for now, will merge regardless